platforms/chibios/bootloaders/none.c

```diff
@@ -19,5 +19,6 @@
 #pragma message "Unknown bootloader set, you may not be able to enter bootloader using software reset"
 
 __attribute__((weak)) void bootloader_jump(void) {}
+__attribute__((weak)) void keyboard_jump(void) { NVIC_SystemReset(); }
```

Comment:
This might have to remain empty, or use `#ifdef NVIC_SystemReset`, if we want to keep this file buildable for any weird MCU (or even not so weird — GD32V does not have that function, although normally it should use its own file).